README.md

```diff
@@ -1372,6 +1372,81 @@ app using the `setUnseenNotificationsBadge`. This way, the next time the webview
 use the getter, it will know if the `lastUpdated` matches with the one persisted
 in `localStorage`.
 
+### requestDatamobDeviceAdmin
+
+<kbd>App version >=24.12</kbd>
+
+Datamob is a native library that offer developers a way to integrate security
+and remote device control features into their applications.
+
+The application that implements the Datamob library must be registered as a
+system management application (Device Admin). This configuration is essential to
+allow the application to have sufficient permissions to execute security
+commands, such as screen lock and factory reset.
+
+This method opens a setting screen asking the user to accept system management
+permissions for the application.
+
+```ts
+requestDatamobDeviceAdmin: () => Promise<{isAdmin: boolean}>;
+```
+
+`isAdmin` is true if the permission was granted.
+
+#### Demo
+
+https://github.com/user-attachments/assets/28095f42-76db-4ac2-9586-e350acef7e1d
+
+### registerDatamobUser
+
+<kbd>App version >=24.12</kbd>
+
+The application that implements the Datamob should have an user registered. This
+method is used to register one.
+
+```ts
+registerDatamobUser: ({phoneNumber: string, tokenPassword: string}) => Promise<{success: boolean}>;
+```
+
+-   `phoneNumber`: The phone number of the user.
+-   `tokenPassword`: When registering the device, datamob generate an accessKey
+    that is recorded in the Datamob device registry. By combining this attribute
+    with a hash that we keep in a password vault, generate this token.
+
+-   `success`: true if the user was registered successfully.
+
+### validateDatamobRequirements
+
+<kbd>App version >=24.12</kbd>
+
+Datamob sdk allows to send remote commands to the user device. These remote
+commands include actions such as locking the device screen (lock screen) or even
+forcing a wipe (factory reset) of the device, providing additional security
+control for the end user.
+
+This method returns a map with the requirements. Each requirement is a boolean
+value where true is valid, false is not valid.
+
+```ts
+validateDatamobRequirements: ({phoneNumber: string, tokenPassword: string}) => Promise<{
+    requirements: {
+        deviceAdmin: boolean;
+        googleAccount: boolean;
+        lockPassword: boolean;
+        accessibilityOption: boolean;
+        invalidPassword: boolean;
```

Comment:
This property was wrong in the issue:

https://github.com/Telefonica/webview-bridge/issues/172

and it has been updated, it should be:

invalidPhoneNumber